### DISCUSSION TITLE
1.3.4 dont work

### DISCUSSION THREAD

**Discussion by sergeenkov73** [Chia-Network/chia-blockchain#11234] [Support]
Good afternoon.

The client of the latest version 1.3.4 does not work.
The “no connection” status disappears when the wallet is restarted, when I enter the mnemonic phrase again, nothing happens. The wallet is not connected. At the same time, the previous version 1.3.3 works without problems.

Добрый день.

Клиент последней версии 1.3.4 не работает.
Статус «нет подключения» при повторном запуске исчезает кошелёк, когда ввожу мнемоническую фразу заново ничего не происходит. Кошелек не подключается. При этом прошлая версия 1.3.3 работает без проблем.

**Comment by loppefaaret** [comment]
If you wait some time, does the "No Connection" disappear ? or does it stays in that same "No Connection" ? 
What OS are you using ? 
Are you running wallet only mode, or a full node / farming mode ?
